x-pack/plugins/rac/scripts/generate_mapping/mappings.js

```diff
@@ -0,0 +1,440 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+exports.EcsKibanaExtensionsMappings = {
+  properties: {
+    signal: {
+      properties: {
+        _meta: {
+          properties: {
+            version: {
+              type: 'long',
+            },
+          },
+        },
+        parent: {
+          properties: {
+            rule: {
+              type: 'keyword',
+            },
+            index: {
+              type: 'keyword',
+            },
+            id: {
+              type: 'keyword',
+            },
+            type: {
+              type: 'keyword',
+            },
+            depth: {
+              type: 'long',
+            },
+          },
+        },
+        parents: {
+          properties: {
+            rule: {
+              type: 'keyword',
+            },
+            index: {
+              type: 'keyword',
+            },
+            id: {
+              type: 'keyword',
+            },
+            type: {
+              type: 'keyword',
+            },
+            depth: {
+              type: 'long',
+            },
+          },
+        },
+        ancestors: {
```

Comment:
same question re: `parents` - should this be `nested`?  Could there be multiple in here?